src/services/preference/preference.service.ts

```diff
@@ -0,0 +1,54 @@
+import { BehaviorSubject, combineLatest, Observable, ObservedValueOf } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AudificationPreference, DataTablePreference, TextSummaryPreference } from './types';
+
+type ObservableDictionary = { [key: string]: Observable<any> };
+type ObservedDictionaryOf<T extends ObservableDictionary> = { [key in keyof T]: ObservedValueOf<T[key]> };
+
+export class PreferenceService {
+  audification = { // mimic namespace
+    enabled: new BehaviorSubject(true),
+    lowestPitch: new BehaviorSubject(256),
+    highestPitch: new BehaviorSubject(1024),
+    noteDuration: new BehaviorSubject(.167),
```

Comment:
should be `const` values

Reply:
you mean making the properties non-observable, or replacing the hard-coded default values with `const` variables?